invenio_app_ils/ui/backoffice/src/pages/ItemDetails/state/types.js

```diff
@@ -1,3 +1,3 @@
-export const IS_LOADING = 'IS_LOADING';
-export const ITEM_DETAILS = 'ITEM_DETAILS';
-export const ITEM_DETAILS_HAS_ERROR = 'ITEM_DETAILS_HAS_ERROR';
+export const IS_LOADING = 'item details/IS_LOADING';
```

Comment:
Prefixing the actions with the actual caller component looks clean and intuitive.
About the prefix though I think I would go for `ItemDetails` as its class or any other pre-agreed combination of camel or snake case, but the space there is killing me.

Reply:
I knew it! I actually commented with @zzacharo saying: "I am sure that if I put a space Chris won't be happy!" but I played my chances :)
I can change it if preferred. (@kprzerwa)

Reply:
I also think we should change the space 😉

Reply:
fixed